docs/faqs.md

```diff
@@ -97,11 +97,19 @@ No, recipes do not need to export a recipe license. Recipes and all files contri
 We generally consider tools like CMake as a standard tool to have installed in your system. Having the `cmake` package as a build require in **all** the recipes that use it will be an overkill, as every build requirement is installed like a requirement and takes time to download. However, `cmake` could still be useful to use in your profile:
 
 ```
-[build_requires]
+[tool_requires]
 cmake/3.17.2
 ```
 
-Other packages using more unusual build tools, like `OpenSSL` using `strawberryperl`, will have the build require in the recipe as it is likely that the user that want to build it from sources will not have it installed in their system
+Other packages using more unusual build tools, like [`OpenSSL`](https://github.com/conan-io/conan-center-index/tree/master/recipes/openssl) using [`strawberryperl`](https://github.com/conan-io/conan-center-index/tree/master/recipes/strawberryperl), will have the build require in the recipe as it is likely that the user that want to build it from sources will not have it installed in their system.
+
+## How are rare build systems without generators packaged?
+
+The C++ ecosystem has a lot of rare, unique and obscure build systems. Some of these are available in ConanCenter but they do not have built-in generators from the main Conan client.
+For these cases, contributors are asked to help reviewers as much as possible as it's likely we will not have expertise.
+
+For quality assurance the build service is expected to be green and the [hooks](https://github.com/conan-io/hooks) will ensure the package contents match what is expected given the options. This means they are more likely to have
```

Comment:
somehow my question got erased from the diff...
who are `they` here? hooks? package contents? options? build services?